### DISCUSSION TITLE
macOS config directory

### DISCUSSION THREAD

**Discussion by whkoh** [KenKundert/emborg#65] [Q&A]
on macOS, is it possible to use `~/.config/emborg` as the config directory instead of `~/Library/Application Support/emborg`?

With config files in `~/.config/emborg`, running `emborg` automatically creates `~/Library/Application Support/emborg` instead of reading from ``~/.config/emborg`.

**Comment by whkoh** [comment]
I ended up creating a symlink to Application Support but would be curious to know either way.

**Comment by KenKundert** [comment]
I am not familiar with how people use MacOS.  Is it common for people to avoid using the Apple selected directory for config files and use the Linux preferred path instead?

**Reply by whkoh** [reply]
It’s quite common - fish, tmux etc are all in `.config`. 

I prefer `.config` as it’s cleaner, easier to backup or use version control rather than the native path. `~/Library/Application Support` has lots of weird locks that borg cannot access during a backup.

**Comment by KenKundert** [comment]
I can check for the existence of ~/.config/emborg and use it if it exists. Give me some time.

**Comment by KenKundert** [comment]
I have upgraded the version on github to support settings files in ~/.config/emborg if it exists.  I cannot test this feature on a Mac as I do not have one.  I would appreciate it if you could try it out and let me know if there are any issues. 

Should I be doing something similar for the directory that holds the Emborg logfiles (~/.local/share/emborg)?
